File: Android App/app/src/main/java/com/makuhita/StockPredAI/entities/Stock.kt
package com.makuhita.StockPredAI.entities

class Stock {
    var symbol: String = ""
    var price: Double = 0.0
    var estimate: Double = 0.0
    var prices: List<Price> = emptyList()
    override fun toString(): String {
        return "Stock(symbol='$symbol', price=$price, estimate=$estimate, prices=$prices)"
    }
}

File: Python Backend/entities/Stock.py
class Stock:
    def __init__(self, symbol, price, estimate, prices):
        self.symbol = symbol
        self.price = price
        self.estimate = estimate
        self.prices = prices

    def to_dict(self):
        return {'symbol': self.symbol, 'price': self.price, 'estimate': self.estimate,
                'prices': self.prices}
